contracts/modules/cover/CoverProducts.sol

```diff
@@ -0,0 +1,224 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.18;
+
+import "@openzeppelin/contracts-v4/security/ReentrancyGuard.sol";
+import "@openzeppelin/contracts-v4/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts-v4/token/ERC20/utils/SafeERC20.sol";
+
+import "../../abstract/MasterAwareV2.sol";
+import "../../abstract/Multicall.sol";
+import "../../interfaces/ICover.sol";
+import "../../interfaces/ICoverNFT.sol";
+import "../../interfaces/IPool.sol";
+import "../../interfaces/IStakingNFT.sol";
+import "../../interfaces/IStakingPool.sol";
+import "../../interfaces/IStakingPoolBeacon.sol";
+import "../../interfaces/IStakingPoolFactory.sol";
+import "../../interfaces/ITokenController.sol";
+import "../../libraries/Math.sol";
+import "../../libraries/SafeUintCast.sol";
+import "../../libraries/StakingPoolLibrary.sol";
+import "../../interfaces/IStakingProducts.sol";
+import "../../interfaces/ICoverProducts.sol";
+
+contract CoverProducts is ICoverProducts, MasterAwareV2, Multicall {
+
+  /* ========== STATE VARIABLES ========== */
+
+  Product[] internal _products;
+  ProductType[] internal _productTypes;
+
+  // productId => product name
+  mapping(uint => string) public productNames;
+  // productTypeId => productType name
+  mapping(uint => string) public productTypeNames;
+
+  // product id => allowed pool ids
+  mapping(uint => uint[]) public allowedPools;
+
+  /* ========== CONSTANTS ========== */
+
+  uint public constant GLOBAL_MIN_PRICE_RATIO = 100; // 1%
```

Comment:
This is also present in the Cover contract. Need to keep it in a single place. There is 1 usage here and 2 usages there, but likely to move one of the two here as well. Decide where to keep this constant and remove it on the other side.